citesphere/src/main/java/edu/asu/diging/citesphere/core/service/impl/CitationManager.java

```diff
@@ -494,4 +494,13 @@ public CitationPage getPrevAndNextCitation(IUser user, String groupId, String co
     public void deleteLocalGroupCitations(String groupId) {
         citationStore.deleteCitationByGroupId(groupId);
     }
+    
+    @Override
+    public CitationResults getCitationsByContributorUri(List<String> groupIds, long start, int pageSize, String uri) {
+    	List<ICitation> citations = (List<ICitation>) citationDao.findCitationsByContributorUri(groupIds, start, pageSize, uri);
+    	CitationResults results = new CitationResults();
+    	results.setCitations(citations);
+    	results.setTotalResults(citations.size());
```

Comment:
that's wrong, isn't it? This is just the total of the returned citations not how many there are total.